test/system/archive_baseline.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh -f
+
+echo
+
+if [ $# -ne 1 ]; then
+    echo "Invoke archive_baseline.sh -help for usage."
+    exit 1
+fi
+
+if [ $1 == "-help" ] || [ $1 == "--help" ]; then
+cat << EOF1
+NAME
+
+	archive_baseline.sh - archive pretag baselines to set locations on
+                              izumi and derecho.
+
+
+SYNOPSIS
+
+	archive_baseline.sh TAGNAME
+	  [-help]
+
+
+ENVIROMENT VARIABLES
+
+	CESM_TESTDIR - Directory that contains the CESM finished results you wish to archive.
+	CAM_FC      - Compiler used, only used on izumi (GNU,NAG), where the compiler
```

Comment:
Will we need to do this for Derecho as well given that we'll have GNU and Intel tests there as well?

Reply:
Yup.  Adding the intel test was a last minute change, and I didn't think about this ramification.